docs/extensions/extension_example.md

```diff
@@ -108,9 +108,9 @@ The lockfile object will look something like this:
 
 ```text
   packages: [
-    { name: "accepts", version: "1.3.8" },
-    { name: "array-flatten", version: "1.1.1" },
-    { name: "accepts", version: "1.0.0" }
+    { lockfile: "package-lock.json", name: "accepts", version: "1.3.8" },
+    { lockfile: "package-lock.json", name: "array-flatten", version: "1.1.1" },
+    { lockfile: "package-lock.json", name: "accepts", version: "1.0.0" }
```

Comment:
```suggestion
    { lockfile: "package-lock.json", name: "accepts", version: "1.3.8", type: "npm" },
    { lockfile: "package-lock.json", name: "array-flatten", version: "1.1.1", type: "npm" },
    { lockfile: "package-lock.json", name: "accepts", version: "1.0.0", type: "npm" }
```

The `type` field must have been added at some point since the document was first written.